galter_subjects_utils/contrib/lcsh/scheme.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 Northwestern University.
+#
+# galter-subjects-utils is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""LCSH Scheme."""
+
+from galter_subjects_utils.scheme import Scheme
+
+
+class LCSHScheme(Scheme):
+    """LCSH subjet kind."""
```

Comment:
Typo from earlier, must have been a copy paste thing.

Reply:
Thanks! I changed "subjet kind" to scheme for simplicity.